Sources/OptionDescriptor.swift

```diff
@@ -586,7 +586,7 @@ struct _Descriptors {
     let wrapReturnType = OptionDescriptor(
         argumentName: "wrapreturntype",
         displayName: "Wrap Return Type",
-        help: "Wrap return type: \"if-multiline\", \"preserve\" (default)",
+        help: "Wrap return type: \"if-multiline\", \"preserve\" (default), \"never\"",
```

Comment:
This is over the 80 character limit (it's 82 chars right now), would suggest doing:

```suggestion
        help: "Wrap return type: \"if-multiline\", \"preserve\", \"never\"",
```